src/ServiceControl/ExternalIntegrations/MessageFailureResolvedByRetryPublisher.cs

```diff
@@ -0,0 +1,35 @@
+namespace ServiceControl.ExternalIntegrations
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Contracts.MessageFailures;
+    using Raven.Client;
+
+    class MessageFailureResolvedByRetryPublisher : EventPublisher<MessageFailureResolvedByRetry, MessageFailureResolvedByRetryPublisher.DispatchContext>
+    {
+        protected override DispatchContext CreateDispatchRequest(MessageFailureResolvedByRetry @event)
+        {
+            return new DispatchContext
+            {
+                FailedMessageId = new Guid(@event.FailedMessageId)
```

Comment:
Was there a reason to convert this to a Guid only to then convert it back to a string when we publish it? Raven is likely going to store it as a json string anyway.

Reply:
Not that I can think of, I'll check if I can leave it as string :-)